tests/services/updater_test.py

```
@@ -0,0 +1,214 @@
+        with tempfile.TemporaryDirectory() as td:
+            tar_path = Path(td) / "evil.tar.gz"
+            with tarfile.open(tar_path, "w:gz") as tar:
+                data = b"pwn"
+                info = tarfile.TarInfo(name="../../evil.txt")
+                info.size = len(data)
+                tar.addfile(info, io.BytesIO(data))
+
+            dest = Path(td) / "out"
+            dest.mkdir()
+            with tarfile.open(tar_path, "r:gz") as tar:
+                with self.assertRaises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)